### ISSUE TITLE
Is it possible to build using numpy > 1.11.* on windows?

### ISSUE THREAD

**Issue by KPLauritzen** [conda-forge/dlib-feedstock#15]
Thank you for maintaining this!

I'm trying to install dlib on windows, with Python 3.6. 
Some other requirements of my project have installed numpy version 1.14.1, and this blocks the installation of `dlib`. 
Trying `conda install -c conda-forge dlib==19.9` gives the following error:

```
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - defaults/win-64::numba==0.36.2=np114py36h12cb543_0
  - dlib==19.9
```

I see in https://github.com/conda-forge/dlib-feedstock/blob/master/recipe/meta.yaml#L36 that numpy is pinned to 1.11 for python3.6 and windows. Is there a specific reason for this?

**Comment by aldanor** [comment]
As of 19.9, looks like it's pinned to the ancient numpy 1.8.* on Linux and OS X as well (https://anaconda.org/conda-forge/dlib/files).

Can it just be updated to 1.14.* on all platforms?

**Comment by patricksnape** [comment]
Great catch on my copy paste error - thanks @aldanor  - fixed by @aldanor in #17
